src/layers/transform/slice.cu

```diff
@@ -276,16 +275,13 @@ void fp_compute_impl(
   pos += sizeof(dim4) * output_strides_list.size();
 
   // Copy tensor data to GPU
-  hydrogen::simple_buffer<unsigned char, El::Device::GPU> device_workspace(
-    l.m_workspace.size(),
-    sync_info);
-  unsigned char* device_workspace_ptr = device_workspace.data();
-  cudaMemcpyAsync(device_workspace_ptr,
-                  l.m_workspace.data(),
-                  l.m_workspace.size(),
-                  cudaMemcpyHostToDevice,
-                  stream);
-  l.m_workspace_event.record(stream);
+  gpu_lib::thrust::vector<unsigned char> device_workspace(l.m_workspace.size());
```

Comment:
Prefer `hydrogen::simple_buffer` to `thrust::vector`.

Reply:
addressed with ba62dcb06925539311074dd079b5e52ff46524ff